### PR TITLE
Create NOTEBOOKS.md

### PR DESCRIPTION
**Pull request by KWeatherwalks**
This request adds a markdown file for keeping track of files added to this folder so data scientists viewing this repo can quickly gain an understanding of what work has been explored and when that work was done.

### CODE REVIEW

**Review by Daniel-Fernandez-951 — APPROVED** [review]
Great idea to organize all the ancillary tools for this project.

**Review by JenFaith — APPROVED** [review]
This is really helpful for keeping everything organized! Love it!

**Comment by kriegersaurusrex** [comment]
Added some updates to notebooks I committed.

**Review by brett-doffing — APPROVED** [review]
Should be quite beneficial to have notebooks documented!

File: notebooks/NOTEBOOKS.md [hunk 1]
```diff
@@ -22,10 +22,10 @@ This folder contains `.ipynb` notebooks and `.py` scripts created by team member
 ## Scripts
 | Filename                      | Created | Description | Dependencies                                  |
 |-------------------------------|---------|-------------|-----------------------------------------------|
-| loop_all_cases.py             | Labs 33 |             | `spacy`                                       |
-| json_to_txt_conversion.py     | Labs 33 |             | `bs4`                                         |
-| cPython_wrapper.py            | Labs 33 |             | `cProfile`                                    |
-| GCP_pdf_to_json_conversion.py | Labs 33 |             | `google.cloud.vision`, `google.cloud.storage` |
+| loop_all_cases.py             | Labs 33 | Function that allows looping over the .txt files of documents stored locally for testing. | `spacy`                                       |
```
Comment (KWeatherwalks) [inline notebooks/NOTEBOOKS.md:25]: Excellent! Thank you!!!

**Pull request merged**